script.module.ws4py/addon.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.ws4py"
+       name="ws4py"
+       version="0.4.3"
+       provider-name="Sylvain Hellegouarch">
+  <requires>
+    <import addon="xbmc.python" version="3.0.0"/>
+  </requires>
+  <extension point="xbmc.python.module"
+             library="lib" />
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en_GB">WebSocket client and server library for Python 2 and 3 as well as PyPy.</summary>
+    <description lang="en_GB">Packed for KODI from https://pypi.org/project/urllib3/</description>
+    <license>BSD</license>
```

Comment:
Please use the correct spdx identifier for the license : https://spdx.org/licenses/